README.md

```diff
@@ -77,6 +77,8 @@ It's the end!
 
 _Spoken languages:_
 
+- Portuguese
```

Comment:
This should be underneath Spanish on the list

Reply:
No

Reply:
I have le power